build.gradle

```diff
@@ -1,50 +1,54 @@
 plugins {
-    id 'java'
     id "org.springframework.boot" version "2.4.1" apply false
+    id "io.spring.dependency-management" version "1.0.10.RELEASE" apply false
     id "org.springdoc.openapi-gradle-plugin" version "1.3.0"
     id "com.github.johnrengelman.processes" version "0.5.0"
 }
 
 allprojects {
     apply plugin: 'io.spring.dependency-management'
+    apply plugin: 'java'
 
     repositories {
         jcenter()
-        mavenCentral()
+        maven { url 'https://jitpack.io' }
+    }
+
+    bootJar {
+        enabled = false
     }
 
     dependencyManagement {
         overriddenByDependencies = false
 
+        ext {
+            openApiVersion = '1.5.2'
```

Comment:
can you explain the need of this var? That would block dependabot from updating it

Reply:
sure, it's for referring to the same openApi version from different `build.gradle` files

Reply:
You have dependency management exactly for that :) also you could go with a native Gradle way - with a platform dep